java/parser/src/test/java/org/sqlflow/parser/ParserAdaptorTest.java

```diff
@@ -0,0 +1,40 @@
+package org.sqlflow.parser;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+
+public class ParserAdaptorTest {
+  @Test
+  public void testCmd() {
+    try {
+      // FIXME(tony): create file in a temporary directory
+      FileUtils.writeByteArrayToFile(new File("test.sql"), "select 1".getBytes());
```

Comment:
Can use [createTempFile](https://docs.oracle.com/javase/7/docs/api/java/io/File.html#createTempFile%28java.lang.String,%20java.lang.String,%20java.io.File%29) to create the temporary file.

Reply:
Thanks, I will try it out in future PRs.